src/components/TextField/tests/TextField.test.tsx

```diff
@@ -344,6 +387,49 @@ describe('<TextField />', () => {
       expect(textField.find(`#${labels[0]}`).text()).toBe('TextField');
       expect(textField.find(`#${labels[1]}`).text()).toBe('kg');
     });
+
+    it('does not set focus `onClick` for the <input /> if the `target` is the `suffix`', () => {
+      const onClickSpy = jest.fn();
+      const mockButtonId = 'MockSuffix';
+      const mockSuffixButton = (
+        <button id={mockButtonId} onClick={onClickSpy} />
+      );
+      const textField = mountWithAppProvider(
+        <TextField
+          label="TextField"
+          suffix={mockSuffixButton}
+          onChange={noop}
+        />,
+      );
+
+      textField.find(`#${mockButtonId}`).simulate('click');
+
+      expect(onClickSpy).toHaveBeenCalled();
+      expect(textField.getDOMNode().querySelector('input')).not.toBe(
+        document.activeElement,
+      );
+    });
+
+    it.only('does not set focus `onFocus` for the <input /> if the `target` is the `suffix`', () => {
```

Comment:
```suggestion
    it('does not set focus `onFocus` for the <input /> if the `target` is the `suffix`', () => {
```

As you pointed out I forgot to remove this 😄